src/test/java/mpack/Assignement3.java

```diff
@@ -0,0 +1,33 @@
+// Links and attributes
+package mpack;
+
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+public class Assignement3 {
+	public static void main(String[] args)
+	{
```

Comment:
If possible move this to base class and call it from there instead of writing it in each class .This will reduce common /duplicate code

Reply:
covered in assignement 1 comment